README.md

```diff
@@ -8,13 +8,26 @@ Prerequisites:
   * [Vagrant](https://www.vagrantup.com/)
   * [Virtualbox](https://www.vagrantup.com/)
   * [Ansible](http://www.ansible.com/) (if using mac w/o root install with `brew install ansible`, otherwise use pip)
+  * you will need to check out both this repo and the cfpb/kratos repo as siblings in the filesystem.
 
-1. add secret settings to `devops/devops/settings_secret.py`
+1. add Node secret setting to `../kratos/src/config_secret.iced`
+1. add Django secret settings to `./devops/devops/settings_secret.py`
 1. `vagrant up`
 1. `vagrant ssh`
-1. `cd /vagrant`
-1. `./manage.py syncdb --noinput`
-1. `./manage.py runserver 0.0.0.0:8000`
+1. load data from github:
+  1. `cd /opt/kratos`
+  1. `icake -n devdesign import_from_gh`
+1. start the django server
+  1. `/vagrant/devops/manage.py runserver 0.0.0.0:8000`
```

Comment:
You added gunicorn to your requirements.txt.  Is that in use yet (along with something like supervisor), or is that still a work in progress hence the need to launch runserver in the manual steps?